src/libasr/pass/replace_symbolic.cpp

```diff
@@ -847,9 +849,10 @@ class ReplaceSymbolicVisitor : public PassUtils::PassVisitor<ReplaceSymbolicVisi
                 } else {
                     print_tmp.push_back(val);
                 }
-            } else {
-                print_tmp.push_back(x.m_values[i]);
-            }
+            } 
+            // else {
+            //     print_tmp.push_back(x.m_values[i]);
```

Comment:
Why is this removed @kmr-srbh ?

Reply:
Oh no! My bad. It must have got uncommented by mistake. I am reverting it.